src/Octavian.jl

```diff
@@ -2,12 +2,14 @@ module Octavian
 
 using VectorizationBase, ArrayInterface, LoopVectorization
 
+using LoopVectorization: PtrArray
+
 using VectorizationBase: align, AbstractStridedPointer, zstridedpointer,
     static_sizeof, lazymul, StridedPointer, gesp, pause, pick_vector_width, has_feature,
     num_cache_levels, cache_size, num_cores, num_cores, cache_inclusive, cache_linesize, ifelse
 using LoopVectorization: maybestaticsize, matmul_params, preserve_buffer, CloseOpen
 using ArrayInterface: OptionallyStaticUnitRange, size, strides, offsets, indices,
-    static_length, static_first, static_last, axes, dense_dims, stride_rank
+    static_length, static_first, static_last, axes, dense_dims, stride_rank, IfElse
```

Comment:
`VectorizationBase.ifelse === IfElse.ifelse`
Perhaps Octavian should have an explicit dependency on [IfElse.jl](https://github.com/SciML/IfElse.jl).

Reply:
Whoops, I didn't see that it was already imported. I'll fix that thanks